### ISSUE TITLE
error: events.googleServiceEmail: must be of type String

### ISSUE THREAD

**Issue by straumat** [SC5/mozaik-ext-sheets#2]
I created a dashboard project following 

``` javascript
npm install -g yo gulp generator-mozaik
yo mozaik
npm install
gulp publish
node app.js
```

I choosed to only require `javascript mozaik-ext-sheets`

I changed config.js to try this

``` javascript
    api: {
        sheets: {
                googleServiceEmail: 'toto',
                googleServiceKeypath: 'tata'
        }
        },
```

No matter what I do, when i try node app.js, i always get : 

```
error: events.googleServiceEmail: must be of type String
events.googleServiceKeypath: must be of type String
```

**Comment by jtbonhomme** [comment]
I proposed a PR #3 to fix this.
In the meantime you can assign a 'foo' value to the default key in the src/config.js.

**Comment by juhamust** [comment]
Oh, sorry the terrible reaction time: I did not have the notifications enabled :( Will have look on this soonish. Thanks for reporting the issue.

**Comment by juhamust** [comment]
The issue is now fixed with release v0.3.0: The actual root cause was the incorrect context `events.<key>` vs `sheets.<key>`
